src/Tools/CoreFx.Tools/GenerateConfigurationProps.cs

```diff
@@ -101,6 +101,20 @@ private void ParseProperties(string propertyName, ProjectRootElement project, bo
                     AddProperties(whenPropertiesElement, value, includeAdditionalProperites, parsedValuePrefix);
                 }
 
+                if (property.Insignificant)
```

Comment:
See https://github.com/dotnet/corefx/blob/master/src/Tools/GenerateProps/properties.props#L26

Naming is hard.  Wes and I spent some time trying to come up with this terminology.